.github/workflows/deploy.yml

```diff
@@ -0,0 +1,107 @@
+name: Deploy to production
+
+on:
+  pull_request:
+    branches:
+      - releases/v1
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
+
+env:
+  # Required for action steebchen/kubectl
+  KUBE_CONFIG_DATA: ${{ secrets.kubeconfig_data_prod }}
+
+jobs:
+  show-changes:
+    if: github.event.pull_request.state != 'closed'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: azure/setup-kubectl@v1
+        with:
+          version: v1.18.0
+
+      - name: Get resource diffs
+        run: |
+          set -e
+          comment_file=/tmp/k8s_diff.txt
+          app_manifest=k8s/app.yaml
+          prod_manifest_dir=k8s.prod
+
+          test -e ~/.kube || mkdir ~/.kube
+          cat <<EOF > ~/.kube/config
+          $(printf '%s' "$KUBE_CONFIG_DATA" | base64 -d)
+          EOF
+
+          app_diff=$(kubectl diff -f $app_manifest || true)
+          if [ -n "$app_diff" ]; then
+            app_diff_msg=$(cat <<EOF
+          To be applied to prod from $app_manifest:
+          \`\`\`
+          $app_diff
+          \`\`\`
```

Comment:
```suggestion
          \`\`\`diff
          $app_diff
          \`\`\`
```

Reply:
this works! see https://github.com/lottspot/chime/pull/15#issuecomment-605704465